src/lib/data.nix

```diff
@@ -11,6 +11,126 @@ let
     emacs -Q --script ${styx-support} --file "$2" -f $1
   '';
 
+  styx-support = pkgs.writeText "styx-support.el" ''
```

Comment:
The customization is the most important part in this exporter, since one should be able to include other emacs packages and customize it with elisp. I'd clean up the following config to its bare minimum and then add an extraConfig param that get appended to the end of the file, AND a config param that totally replaces it.

Reply:
Also, I'd move this and it's customization options to a separate nix file. Any naming suggestion?